modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -372,98 +364,265 @@ class RuleList extends Component {
 	}
 }
 
-RuleList.STATE = {
-	dataProvider: Config.arrayOf(
-		Config.shapeOf({
-			id: Config.string(),
-			name: Config.string(),
-			uuid: Config.string(),
-		})
-	),
-
-	dropdownExpandedIndex: Config.number().internal(),
-
-	pages: Config.array().required(),
-
-	roles: Config.arrayOf(
-		Config.shapeOf({
-			id: Config.string(),
-			name: Config.string(),
-		})
-	).value([]),
-
-	/**
-	 * @default 0
-	 * @instance
-	 * @memberof RuleList
-	 * @type {?array}
-	 */
-
-	rules: Config.arrayOf(
-		Config.shapeOf({
-			actions: Config.arrayOf(
-				Config.shapeOf({
-					action: Config.string(),
-					ddmDataProviderInstanceUUID: Config.string(),
-					expression: Config.string(),
-					inputs: Config.object(),
-					label: Config.string(),
-					outputs: Config.object(),
-					target: Config.string(),
-				})
-			),
-			conditions: Config.arrayOf(
-				Config.shapeOf({
-					operands: Config.arrayOf(
-						Config.shapeOf({
-							label: Config.string(),
-							repeatable: Config.bool(),
-							type: Config.string(),
-							value: Config.string(),
-						})
-					),
-					operator: Config.string(),
-				})
-			),
-			['logical-operator']: Config.string(),
-		})
-	).value([]),
-
-	/**
-	 * @default undefined
-	 * @instance
-	 * @memberof RuleList
-	 * @type {!string}
-	 */
-
-	spritemap: Config.string().required(),
-
-	/**
-	 * @default strings
-	 * @instance
-	 * @memberof RuleList
-	 * @type {!object}
-	 */
-
-	strings: Config.object().value({
-		'belongs-to': Liferay.Language.get('belongs-to'),
-		'calculate-field': Liferay.Language.get('calculate-field-x-as-x'),
-		contains: Liferay.Language.get('contains'),
-		'due-to-missing-fields': Liferay.Language.get('due-to-missing-fields'),
-		'equals-to': Liferay.Language.get('is-equal-to'),
-		'greater-than': Liferay.Language.get('is-greater-than'),
-		'greater-than-equals': Liferay.Language.get(
-			'is-greater-than-or-equal-to'
-		),
-		'is-empty': Liferay.Language.get('is-empty'),
-		'less-than': Liferay.Language.get('is-less-than'),
-		'less-than-equals': Liferay.Language.get('is-less-than-or-equal-to'),
-		'not-contains': Liferay.Language.get('does-not-contain'),
-		'not-equals-to': Liferay.Language.get('is-not-equal-to'),
-		'not-is-empty': Liferay.Language.get('is-not-empty'),
-	}),
-};
-
-Soy.register(RuleList, templates);
-
-export default RuleList;
+/**
+ * Prints the DDM form card rule.
+ */
+const EmptyList = ({message}) => {
+    return (
+        <div className="main-content-body">
+            <div className="sheet taglib-empty-result-message">
```

Comment:
Did you already tried [ClaySheet](https://clayui.com/docs/components/layout/api.html#sheet)?